test/tools/fixtures/process_resource_script.in.js

```diff
@@ -78,12 +78,14 @@ async function main() {
 main()
   .then(() => {})
   .catch(e => {
-    log({ exitCode: 1, error: util.inspect(e) });
+    log({ exitCode: 1, error: { message: e.message, stack: e.stack } });
```

Comment:
I just realized we're not setting the exit code here 😅 just so we don't confuse ourselves (and since we're not using it) can you remove `exitCode: 1` here and add:

```js
process.exitCode = 1;
```

Same with the other log line, just remove the `exitCode: 99,` but still invoke `process.exit(99);`. The difference is there we're synchronously requesting the process to exit, here we want to set the code and let it exit because it's out of ref-ed resources.

Reply:
Mine auto-set from the catch, but I'll add in the explicit exitCode = 1 here.